examples/fast-api/fast_api.py

```
@@ -0,0 +1,138 @@
+
+    # Log response
+    await logger.log(
+        "info",
+        f"Response {response.status_code}",
+        {
+            "method": request.method,
+            "path": request.url.path,
+            "status_code": response.status_code,
+            "duration": duration,
+        },
+    )
```

Pylint (3.3.7)

[error] Too many positional arguments for method call

(E1121)